frontend/sw360-portlet/src/main/webapp/css/sw360.css

```diff
@@ -237,4 +238,13 @@ div.content2 {
 
 input[readonly].clickable {
     cursor: pointer;
-}
\ No newline at end of file
+}
+
+#releaseSelect {
+    margin-bottom: -1px;
```

Comment:
There might be a better solution then negative margins. It might be worth to have a look at http://stackoverflow.com/questions/5506844/negative-top-margin-not-working-in-ie-8-or-9